contracts/governance/extensions/GovernorVotingSimple.sol

```diff
@@ -19,63 +19,77 @@ abstract contract GovernorVotingSimple is IGovernor {
         Abstain
     }
 
-    struct Voting {
+    struct Receipt {
```

Comment:
I would not use the term "receipt" for this. Because in Bravo there is one proposal per account (per proposal). Something like `ProposalVotes` sounds more appropriate to me.